src/api/graphql/events/EventCreateMutation.ts

```diff
@@ -43,10 +64,19 @@ export interface EventRequestInput {
     }[]
 }
 
-export interface EventMutationVars {
+export interface CreateEventMutationVars {
     request: EventRequestInput
 }
 
-export interface EventMutationData {
+export interface ReplaceEventMutationVars {
+    id: string,
+    request: EventRequestInput
+}
+
+export interface CreateEventMutationData {
     createEvent: EventData
+}
+
+export interface ReplaceEventMutationData {
```

Comment:
we really need to generate types for typescript to end this hell

Reply:
I added a task for it in trello